src/main/java/dansplugins/factionsystem/placeholders/PlaceholderAPI.java

```diff
@@ -54,15 +55,15 @@ public boolean canRegister() {
 
     @Override
     public String onPlaceholderRequest(Player player, @NotNull String id) {
-        id = id.toLowerCase(); // I'm unsure if PlaceholderAPI enforces case, but lets just do it to make sure.
+        id = id.toLowerCase(); // I'm unsure if PlaceholderAPI enforces case, but let's just do it to make sure.
         if (player == null) return null; // We only want to handle Player-Placeholders here.
 
         final boolean hasFaction = persistentData.isInFaction(player.getUniqueId());
         final Faction faction = persistentData.getPlayersFaction(player.getUniqueId());
 
         // Prerequisites.
         if (id.startsWith("faction_") && !hasFaction && !id.equalsIgnoreCase("faction_at_location")) {
-            return "Factionless"; // We don't want Faction-Specific Placeholders to return if they are Factionless!
+            return new ConfigService().getString("factionless"); // We don't want Faction-Specific Placeholders to return if they are Factionless!
```

Comment:
We should get the config service via dependency injection, not create a new instance each time we need it

Reply:
This is probably the fastest and best way for users